### PR TITLE
English comments and Config from file

### LINKED ISSUE #1: Rewrite comments in English

### LINKED ISSUE #4: Read variables from separate file
Instead of hardcoding them in the script itself.

For the sake of proper version control.

### PR DESCRIPTION
**Pull request by tdltdc**

### CODE REVIEW

File: sbsc.py [hunk 1]
```diff
@@ -15,81 +16,91 @@
...
 import time
+from Configuration import Configuration
+
 
-channel_id = ''
-api_key = ''
+conf = Configuration()
+conf.load_json("./config.json")
 
-# Wacht tot systeemklok via NTP is gesynchroniseerd
```
Comment (dennisvisschers) [inline sbsc.py:22]: no English comment to replace Dutch comment

**Comment by dennisvisschers** [comment]
i'm OK with this, if it can be tested before taking into production

**Comment by dennisvisschers** [comment]
i probably closed the issues too soon; should have let the PR take care of it

**Comment by tdltdc** [comment]
> i'm OK with this, if it can be tested before taking into production

I've tested it with the new configuration file. All seems to be working as before (i.e. well ;).

**Pull request merged**